docs/operator-manual/cluster-bootstrapping.md

```diff
@@ -119,4 +119,29 @@ metadata:
   - resources-finalizer.argocd.argoproj.io
 spec:
  ...
-``` 
+```
+
+### Ignoring some differences in managed applications
+
+If you don't want your app of apps becoming out of sync on a regular basis and/or want to manually modify some configurations for managed applications like auto sync being enabled, you can configure ignoring differences in the app of apps, e.g.
+
+```yaml
+spec:
+  ...
+  syncPolicy:
+    ...
+    syncOptions:
+      - RespectIgnoreDifferences=true
+    ...
+  ignoreDifferences:
+    - group: "*"
+      kind: "Application"
+      namespace: "*"
+      jsonPointers:
+        # To allow manually turn off auto sync for apps and preserve that setting
+        - /spec/syncPolicy/automated
+        # These are automatically updated on a regular basis. Not ignoring last applied configuration since it's used for computing diffs after normalization.
+        - /metadata/annotations/argocd.argoproj.io~1refresh
+        - /operation
```

Comment:
```suggestion
        # Allow manually disabling auto sync for apps, useful for debugging.
        - /spec/syncPolicy/automated
        # These are automatically updated on a regular basis. Not ignoring last applied configuration since it's used for computing diffs after normalization.
        - /metadata/annotations/argocd.argoproj.io~1refresh
        - /operation
```

Reply:
@andrii-korotkov-verkada Can you explain why you ignore the annotations? I wouldn't think this would add anything.

Reply:
@todaywasawesome, I've noticed that some annotations and /operation were constantly triggering out of sync status for the app of apps despite no changes in the manifests, since they could be added/removed/updated by automation. I thought it'll be cleaner to ignore them.